<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <!-- Usuario -->
   <item>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="placeholderText">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <!-- Contraseña -->
   <item>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
   </item>
   <!-- Recordar usuario -->
   <item>
    <widget class="QCheckBox" name="rememberCheckBox">
     <property name="text">
      <string>Recordar usuario</string>
     </property>
    </widget>
   </item>
   <!-- Enlace Registrate Aquí -->
   <item>
    <widget class="QLabel" name="signupLabel">
     <property name="text">
      <string>&lt;a href=&quot;#&quot;&gt;Regístrate aquí&lt;/a&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #2a5db0; font-size: 9pt; text-decoration: underline; cursor: pointer;</string>
     </property>
    </widget>
   </item>
   <!-- Botón Iniciar Sesión -->
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
     <property name="minimumHeight">
      <number>32</number>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 13pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
